### PR TITLE
Create serializer for Health-officer model in serializer.py

### LINKED ISSUE #8: Create HealthOfficer Model Serializer
Create Serializer class for HealthOfficer Model.
Create class in api/serializers.py file.
Class name should be **HealthOfficerSerializer**.

### LINKED ISSUE #12: CRUD Endpoints for HealthOfficer Model
Implement endpoints for Creating, Retrieving, Updating and Deleting Health Officers.
Endpoints should return data in json format.
Implement endpoints in api/views.py file and create the url routes in api/urls.py file.

## New Routes to be Added to api/urls.py file
**GET user/doctor/**  return all doctors **status - 200**
**GET user/doctor/uuid/**  return doctor with uuid **status - 200**
**POST user/doctor/**  create a new Health Officer  **status - 201**
**PATCH user/doctor/uuid/**  Update record of health officer with uuid
**DELETE user/doctor/uuid/**  Delete record of health officer with uuid

### PR DESCRIPTION
**Pull request by blacdev**
Create CRUD for Health-officer serializer in views.py
Create end-point for Health-officer-create-list-view and Health-officer-retrieve-update-view in api/urls.py

### CODE REVIEW

File: api/serializers.py [hunk 1]
```diff
@@ -20,10 +20,26 @@ class PatientSerializer(serializers.ModelSerializer):
...
+
+    class Meta:
+        model = HealthOfficer
+        fields = '__all__'
+
+    def create(self, validated_data):
+        user_data = validated_data.pop('user')
+        user = User.objects.create_user(**user_data)
+        officer = HealthOfficer.objects.create(**validated_data, user=user)
+        patient.save()
```
Comment (austinoski) [inline api/serializers.py]: Why call **patient.save()** here? think it should be **officer.save()**.

File: api/views.py [hunk 2]
```diff
@@ -28,7 +28,7 @@ def post(self, request, format=None):
 
 class PatientRetrieveUpdateAPIView(APIView):
     serializer_class = PatientSerializer
-    permission_classes = [permissions.IsAuthenticated]
+    # permission_classes = [permissions.IsAuthenticated]
```
Comment (austinoski) [inline api/views.py]: **PatientRetrieveUpdateAPIView** should be protected. uncomment **permission_classes** attribute.

File: api/serializers.py (hunk 1 above)
Comment (blacdev) [reply api/serializers.py]: can you see that i have made changes @austinoski

Comment (austinoski) [reply api/serializers.py]: I'll check it now.

**Review by austinoski — APPROVED** [review]

**Pull request merged**